models/issue_comment.go

```diff
@@ -1016,10 +1016,6 @@ func fetchCodeCommentsByReview(e Engine, issue *Issue, currentUser *User, review
 		return nil, err
 	}
 
-	if err := CommentList(comments).loadPosters(e); err != nil {
```

Comment:
Is this deliberate?

Reply:
Yes, it's a duplicate of the same action two lines below (or was it above?)